src/app/components/about/about.ts

```diff
@@ -5,14 +5,7 @@ import {Http} from 'angular2/http';
 @Component({
   selector: 'about',
   templateUrl: 'app/components/about/about.html',
-  styleUrls: ['app/components/about/about.css'],
-  providers: [],
-  directives: [],
-  pipes: []
```

Comment:
This project is show a show off of what angular2 does, so this stuff is just for demonstration purposes.